contributors/guide/scalability-good-practices.md

```diff
@@ -102,68 +96,62 @@ func (c *ControllerX) Run() {
 
 ### Big dependency changes
-*TL;DR:* When updating big dependencies (especially etcd or golang version) you need to run large scale test to verify that bump doesn't break Kubernetes performance.
- 
-Kubernetes depends on pretty much the whole universe. From time to time we need to update some dependencies (Godeps, etcd, go version). This can break us in many ways, and we saw it doing it a couple of times now. We skipped one version of `go` (1.5) exactly because of breaking our performance, and in the moment of writing this doc we're working with go team on understanding why `go1.8` also makes Kubernetes performance noticeably worse.
+Kubernetes depends on pretty much the whole universe. From time to time we need to update some dependencies (Godeps, etcd, go version). This can break us in many ways, as has already happened a couple of times. We skipped one version of `go` (1.5) precisely because it broke our performance. As this is being written, we are working with the go team to try to understand why `go1.8` negatively affects Kubernetes performance.
  
-If you're changing some big and important dependency, there's no way of knowing what performance impact it'll have. The only thing we can do is to actually run test and check.
+If you are changing a large and important dependency, there is no way of knowing what performance impact it will have. We can only run test and check.
 Where to look to get data?
-If you want to check the impact of your changes there are a number of places you can look at:
-Density/Load tests output quite a lot of data either to test logs, or files inside 'ReportDir' - both of them include API call latencies and Density also includes pod e2e startup latency information,
-for resource usage you can either use monitoring tools (heapster + graphana, but at the time of writing it stops working around 100 Nodes), or just plain old top on master (which scales as much as you want:),
-more data is available on `/metrics` endpoint on all our components (e.g. one for the API server contains API call latencies),
-to profile a component is to create an ssh tunnel to the machine running it, and run `go tool pprof localhost:<your_tunnel_port>` locally
+If you want to check the impact of your changes there are a number of places where you can look:
+Density/Load tests output quite a lot of data either to test logs, or files inside 'ReportDir' - both of them include API call latencies, and Density also includes pod e2e startup latency information.
+for resource usage you can either use monitoring tools (heapster + graphana, but at the time we wrote this it stops working at around 100 Nodes), or just plain 'top' on master (which scales as much as you want),
```

Comment:
Capitalize "For"
"Grafana"
"...note that at the time we wrote this, this stops working...."
"control plane node not master"

Reply:
Done.